Misc/NEWS.d/next/Library/2023-04-22-22-37-39.gh-issue-103699.NizCjc.rst

```diff
@@ -0,0 +1,2 @@
+Add ``__orig_bases__`` to non-generic TypedDicts, call-based TypedDicts, and
+call-based NamedTuples
```

Comment:
```suggestion
call-based NamedTuples. Other TypedDicts and NamedTuples already had the attribute.
```
Otherwise readers of the news entry might get worried.